docs/home.md

```diff
@@ -57,93 +84,135 @@ underlying [Marinade Finance protocol](https://docs.marinade.finance/faq/faq#wha
 
 ## The Details
 
-### What is Liquid Staking?
+### A Deep Dive into the Staking Process
+Let’s take a deep dive into what happens when you deposit your SOL with Sunrise Stake
 
-Liquid staking is a method of staking that allows stakers to unstake without
-the typical 2-3 day withdrawal period.
+#### Step 1: Depositing SOL with Sunrise Stake
+The staking process begins with you depositing your SOL via Sunrise Stake’s app.
 
-Sunrise proxies the [Marinade Finance](https://docs.marinade.finance/) protocol,
-which allows liquid staking through the use of liquidity pools.
+Your SOL will be deposited in Sunrise Stake’s Treasury Account, which is controlled by a smart contract (called Treasury Controller). 
+The Treasury Account is a Programme Derived Account (PDA) that is controlled by the Treasury Controller program; this means that only the Treasury Controller program can move the funds in the Treasury Account.
```

Comment:
Program instead of Programme (the US spelling is exclusively used for software programs)

Reply:
Done